bower.json

```diff
@@ -27,7 +27,7 @@
   "homepage": "https://github.com/PolymerElements/paper-input",
   "ignore": [],
   "dependencies": {
-    "polymer": "Polymer/polymer#^1.2.0",
```

Comment:
It's a little weird that a previous version of this element requires v1.2.0, but a more recent version requires v1.1.0.. but, I guess it's not any weirder than the rest of our dependency graph.